src/runner.ts

```diff
@@ -206,22 +249,54 @@ async function addParachainsToGenesis(
 		return { isSimple: true, ...p };
 	});
 	let paras = x.concat(y);
+	let paraChainSpecRawPaths = new Map();
 
 	for (const parachain of paras) {
-		const { resolvedId, chain } = parachain;
-		const bin = resolve(config_dir, parachain.bin);
+		const { isSimple, id, resolvedId, protocolId, chain = "local" } = parachain;
```

Comment:
```suggestion
		const { isSimple, id, resolvedId, protocolId, chain = "local" } = parachain;
```

Where are you getting `chain = "local"` from?

This is breaking the application from running.

Are you testing polkadot-launch after the changes you made?

Reply:
If within the `parachain` section the `chain` value is not specified, then what should its default value be?

Reply:
Also, I tried with one relay chain of 3-node and one parachain of 2-node with config file [listed here](https://gist.github.com/jimmychu0807/7db192e10b1a189e28cfc83eb3bacacf) and works.

But as [mentioned above](https://github.com/paritytech/polkadot-launch/pull/160#issue-1078096354), I cannot make it to work with 3-node relay chain and two 1-node parachain networks.

What test config do you use when testing?

Reply:
Right now I was trying with just the default config, generating a binary from cumulus and polkadot master.

Cumulus `polkadot-collator` node does not have a `local` chain. By default, I don't think it needs a chain input.